tasks/installer.py

```diff
@@ -23,7 +23,8 @@ def build(
     build_include=None,
     build_exclude=None,
     go_mod="mod",
-    no_strip_binary=True,
+    no_strip_binary=False,
```

Comment:
Why is the default value changing? It seems unrelated to the change in itself

Reply:
I removed that change - let's discuss that offline or in another PR